libass/ass.h

```diff
@@ -194,6 +194,10 @@ typedef enum {
      * On dialogue events override: Justify
      */
     ASS_OVERRIDE_BIT_JUSTIFY = 1 << 10,
+    /**
+     * On dialogue events override: Blur
+     */
+    ASS_OVERRIDE_BIT_BLUR = 1 << 11,
```

Comment:
To allow users to guard usage of the new enum, `LIBASS_VERSION` should be bumped too